vendor/data/country_to_region_mapping.rb

```diff
@@ -0,0 +1,253 @@
+﻿REGION_CODES = {

```

Comment:
I wonder if putting this in a 'beis' folder makes the distinction about 'who' did the mapping simpler to access?

Reply:
I've moved it into a BEIS folder 👍  

I still am not sure if this is a very Rails-y way of doing it, maybe a module with a constant in it is better? 🤔

Reply:
When I think about how all the other big lists of data come into the applicaiton, I would look for something like this  not a module - I would probably look for yaml and a code list helper - as that seems to be the pattern we already have.

That's not to say this doesn't work and is more elegant, it's more about how aproachable the code?

Reply:
Yeah I think you're right... I will switch this to YAML for consistency!